ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp-415-quay39.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: quay-e2e-tests-quay39-ocp415-lp-interop
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      CSO_CHANNEL: stable-3.9
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPINTEROP"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP"},
+          {"step": "enable-*", "failure_type": "all", "classification": "Other", "jira_project": "PROJQUAY", "jira_additional_labels": ["interop-tests"]},
+          {"step": "quay-tests-deploy-*", "failure_type": "all", "classification":  "Quay deployment", "jira_project": "PROJQUAY", "jira_additional_labels": ["interop-tests"]},
+          {"step": "quay-tests-*", "failure_type": "all", "classification":  "Test Failure", "jira_project": "PROJQUAY", "jira_additional_labels": ["interop-tests"]}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      ODF_OPERATOR_CHANNEL: stable-4.13
```

Comment:
Is this the correct channel for OCP 4.15?

Reply:
This is what currently exists for testing quay 3.9 on OCP 4.14
https://github.com/openshift/release/blob/master/ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp-414-quay39.yaml#L38

Here's a link to a nice openshift/layered product support page, though 4.15 is not mentioned
https://access.redhat.com/support/policy/updates/openshift

I honestly do not know the answer to this. This might be a case where we present an issue to PQE next week and allow them to correct this.
- The alternative to that would be to pull this scenario from this PR and create an individual PR for the quay scenario where PQE can provide info.